### PR TITLE
IOW-289 changelog update for release

### PR DESCRIPTION
**Pull request by orosz-usgs**
Before making a pull request
----------------------------

- [ ] Make sure all tests run
- [ ] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
If no JIRA ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the JIRA ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html). (Patch version X.Y.0 is implied if not specified.)
 
-## [Unreleased](https://github.com/USGS/time-series-services/compare/time-series-services/compare/0.2.0...master)
+## [Unreleased](https://github.com/USGS/time-series-services/compare/time-series-services/compare/...master)
+
+## [0.3.0](https://github.com/USGS/time-series-services/compare/time-series-services/compare/0.3.0...master)
```
Comment (mbucknell) [inline CHANGELOG.md]: This should compare 0.2.0 to 0.3.0

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,7 +4,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html). (Patch version X.Y.0 is implied if not specified.)
 
-## [Unreleased](https://github.com/USGS/time-series-services/compare/time-series-services/compare/0.2.0...master)
+## [Unreleased](https://github.com/USGS/time-series-services/compare/time-series-services/compare/...master)
```
Comment (mbucknell) [inline CHANGELOG.md]: This should compare 0.3.0 to master. P

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,9 +4,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html). (Patch version X.Y.0 is implied if not specified.)
 
-## [Unreleased](https://github.com/USGS/time-series-services/compare/time-series-services/compare/...master)
+## [Unreleased](https://github.com/USGS/time-series-services/compare/time-series-services/compare/0.3.0...master)
```
Comment (mbucknell) [inline CHANGELOG.md]: So you should replace 0.3.0 with 0.4.0

**Comment by orosz-usgs** [comment]
@dsteinich do you have anything else?

**Pull request merged**